cpp/integ/projector_test.cc

```diff
@@ -433,4 +437,103 @@ TEST_F(TestProjector, TestZeroCopyNegative) {
   status = projector->Evaluate(*in_batch, {bad_array_data3});
   EXPECT_EQ(status.code(), StatusCode::Invalid);
 }
+
+long long MillisSince(time_t base_line, int yy, int mm, int dd, int hr, int min, int sec) {
+  struct tm given_ts = {0};
+  given_ts.tm_year = (yy - 1900);
+  given_ts.tm_mon = (mm - 1);
+  given_ts.tm_mday = dd;
+  given_ts.tm_hour = hr;
+  given_ts.tm_min = min;
+  given_ts.tm_sec = sec;
+
+  return (lround(difftime(mktime(&given_ts), base_line)) * 1000);
+}
+
+TEST_F(TestProjector, TestTime) {
```

Comment:
can you pls add the date/time tests to a new file ? otherwise, this file will get very big.